react/typings/events.d.ts

```diff
@@ -204,11 +204,16 @@ export interface SearchData extends EventData {
 }
 
 export interface LoginData extends EventData {
-  event: 'viewCart'
-  eventType: 'vtex:viewCart'
+  event: 'login'
+  eventType: 'vtex:login'
```

Comment:
👏 

Totally missed that in the last [PR](https://github.com/vtex-apps/google-tag-manager/pull/122/files#diff-791619406f30fb7292cdecb798d2a840d54a926399fdcd740a7706dbe852c9a6)... 😬